mern-ecommerce/controllers/auth.js

```diff
@@ -56,13 +56,13 @@ exports.signout = (req, res) => {
   res.json({ message: 'Signout success' });
 };
 
-exports.requireSignin = expressJwt({
+exports.requireSignisn = expressJwt({
```

Comment:
Typo in the function name. It should be `requireSignin` instead of `requireSignisn`.